provider/core.rb

```diff
@@ -332,9 +332,18 @@ def generate_objects(output_folder, types, version_name)
 
     def generate_object(object, output_folder, version_name)
       data = build_object_data(object, output_folder, version_name)
+      unless object.exclude_resource
+        Google::LOGGER.info "Generating #{object.name} resource"
```

Comment:
Can you move these to LOGGER.debug or lower? Too much granularity at info means people just ignore all of it.